### ISSUE TITLE
Building site locally returns ERROR on google_analytics_async.html

### ISSUE THREAD

**Issue by JMHReif** [theNewDynamic/gohugo-theme-ananke#694]
Hello! I added a new blog post to my site on May 23, 2024 with no issues, but when I try to add another one and test it today, I'm getting the following error.

```
ERROR render of "section" failed: execute of template failed: html/template:_default/list.html:40:18: no such template "_internal/google_analytics_async.html"
ERROR render of "page" failed: execute of template failed: html/template:_default/single.html:40:18: no such template "_internal/google_analytics_async.html"
Built in 143 ms
Error: error building site: render: failed to render pages: render of "home" failed: execute of template failed: html/template:index.html:40:18: no such template "_internal/google_analytics_async.html"
```

I see there was a commit in this theme to fix that problem, but I've installed this theme as a submodule in my repository, and I'm not sure why it's not pulling in the change. I've linked my repository (as it was, working on 05/23/2024). https://github.com/JMHReif/jmhreif-website

Could someone assist with determining what I'm missing? Thank you!

**Comment by ricardopoffo** [comment]
Hello! I cloned your repo and reproduced the problem on my local machine. I got rid of the error message by creating an `_internal/google_analytics_async.html` file. I consider this a temporary solution. I came here looking for help with another problem, but I figured I could help you as well. I have a very similar blog. Could you tell us more about the changes made since the last working version?

**Comment by JMHReif** [comment]
I added the `_internal/google_analytics_async.html` file, but it's still throwing the error. Do I need to add that in a specific place in my repo?

**Comment by JMHReif** [comment]
As far as the changes made, I added a new blog post under the `content` folder, added a new image under the `static` folder (both are things I do all the time). I also updated a few of my old blog post names to start with a date, so that they have a consistent order under the blog folder. I wouldn't think renaming posts would cause that particular error, but maybe?

**Comment by DevTown** [comment]
The Bug is fixt in Main but not in the newest release. :-(

**Comment by JMHReif** [comment]
I'm sorry, @DevTown, my frontend skills are less than stellar.....do I need to copy the whole baseof.html file from the theme repo into my repo and then modify there for it to pick up the change?

**Comment by JMHReif** [comment]
@DevTown and @ricardopoffo, I don't have Google Analytics enabled on my repo. The theme is a submodule in my repo, but I haven't pulled all the files into my repository (folder placeholder only). Could someone explain a bit more how this should work? I would really appreciate a bit more explanation! :)

**Comment by DevTown** [comment]
@regisphilibert Could you deploy a fixed version from the main plz ?

@JMHReif : When you want to create your site local you can go to your checkout theme folder .
Where you go to the ananke\layouts_default Folder .
There you open the file baseof.html  in a editor of your choice .
Now delete line 40 and past the following line in it:

{{ template "_internal/google_analytics.html" . }}

After you save the file you can run hugo again.

The problem is not that you are using google_analytics or not. The theme has the possibility to use it and the theme has a bug.  It is fixed in the githubrepo but not i a released.

**Comment by JMHReif** [comment]
Thank you for the help, @DevTown! My site can be updated again. :)

**Comment by s7nio** [comment]
Identical setup and problem for me. When is the next ananke release planned for this?

**Comment by fredcy** [comment]
To get this theme to work with hugo v0.126 (as installed from nix packages 24.05) I had to revert the theme to the v2.9.2 tag and then apply DevTown's patch above.

**Comment by jmooring** [comment]
This can be closed. Resolved with https://github.com/theNewDynamic/gohugo-theme-ananke/commit/26a78654fabfe49c29db831f74963564175ecc2d. Available in v2.10.0.

**Comment by davidsneighbour** [comment]
@jmooring thank you!
